drivers/at30tse75x/at30tse75x.c

```diff
@@ -0,0 +1,312 @@
+/*
+ * Copyright (C) Daniel Krebs
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @file
+ * @brief       Driver for the AT30TSE75x temperature sensor with serial EEPROM
+ *
+ * @author      Daniel Krebs <github@daniel-krebs.net>
+ */
+
+
+#include "periph/i2c.h"
+#include "hwtimer.h"
```

Comment:
@daniel-k, sorry for the delay. Could you adapt the implementation to use `xtimer` please